gcp_variant_transforms/libs/call_info_table_schema_generator.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Google LLC.
```

Comment:
In VCF specs, it seems instead of "call_info" they used "samples" or "sample_ids" to refer to them. Do you think it would be more clear if we rename all new modules and classes with one of those two?

Reply:
I definitely would like to see us move to sample over time :)

So naming this call_info now might be an error regardless of what we call it internally (eg renaming the classes can come later)